core/system/src/mpu/vmpu_freescale_k64.c

```diff
@@ -171,46 +164,38 @@ void vmpu_acl_stack(uint8_t box_id, uint32_t context_size, uint32_t stack_size)
     stack_size = UVISOR_REGION_ROUND_UP(UVISOR_MIN_STACK(stack_size));
 
     /* add stack ACL */
-    vmpu_acl_add(
-        box_id,
-        (void*)g_box_mem_pos,
+    vmpu_acl_add(box_id,
+        (void *) g_box_mem_pos,
```

Comment:
This should be indented below `box_id`, similarly to other changes here introduced